erica/api/v2/endpoints/est.py

```diff
@@ -2,21 +2,26 @@
 from uuid import UUID
 
 from fastapi import status, APIRouter
+from opyoid import Injector
 from starlette.responses import JSONResponse, RedirectResponse
 
-from erica.api.utils import generate_error_response, get_erica_request, map_status, get_entity_not_found_log_message
-from erica.api.v2.responses.model import response_model_get_send_est_from_queue, JobState, EstResponseDto, \
-    ResultGetSendEstFromQueue, ErrorRequestQueue
+from erica.api.ApiModule import ApiModule
+from erica.api.utils import generate_error_response, get_entity_not_found_log_message
+from erica.api.v2.responses.model import response_model_get_send_est_from_queue, response_model_post_to_queue
 from erica.application.JobService.job_service_factory import get_job_service
+from erica.application.tax_declaration.TaxDeclarationService import TaxDeclarationServiceInterface
 from erica.application.tax_declaration.tax_declaration_dto import TaxDeclarationDto
 from erica.domain.Shared.EricaRequest import RequestType
 from erica.infrastructure.sqlalchemy.repositories.base_repository import EntityNotFoundError
 
 router = APIRouter()
 
+injector = Injector([
+    ApiModule(),
+])
```

Comment:
I am not the biggest fan of that. Because if we hardcode it, we do not really have a possibility to change it for e.g. the tests and then have to go back to mocking. I am wondering if there is a way to get that from somewhere else (s. also the comment from Sanny)